test/test_plugin_manager.rb

```diff
@@ -110,6 +110,10 @@ def with_no_gemfile
   end
 
   context "plugins_dir is set to a different dir" do
+    setup do
+      skip_if_windows "Jekyll doesn't currently support " \
+      "setting plugins_dir to a different dir on Windows"
```

Comment:
If we don't support it, then this should be caught in the application code, not told in the test code.